src/translations/zh_tw.json

```diff
@@ -2619,11 +2630,14 @@
     "-423203411": "1. <0>登入</0>到您的Deriv帳戶。沒有帳戶？免費<1>註冊</1>。",
     "-1979013933": "2. 前往收銀台 > Deriv P2P 及註冊。",
     "-1719881739": "在您的手機上",
+    "-1390185987": "1. 下載 Deriv P2P。",
     "-293389307": "完成交易",
+    "-2138310324": "掃描二維碼下載Deriv P2P",
     "-1245597945": "立刻開始試用 Deriv P2P",
+    "-834818166": "21,000+",
     "-1154771259": "想了解更多關於 Deriv P2P 的資訊？前往我們的<0>幫助中心</0>",
     "-385361251": "節省時間",
-    "-1007066865": "數分鐘內兌換。<0 />等待時間更短，交易更多。<0 />數分鐘內完成存取款。",
```

Comment:
Is this a mistake from translation side? Can we double check?

Reply:
I checked with Putri and she updated it 🙏